### PR TITLE
Initial task to republish all forms

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/lvTJTH39/945-rake-task-to-republish-all-forms-1-hour)

## Context

Added a script that re-publishes all the forms in publisher.

The criteria to be published:

1. There are latest completed deployments
2. The service is running

To run:

```
USER=my-user@myemail.com bundle exec rake republish_all_forms
```

### CODE REVIEW

File: lib/tasks/republish.rake [hunk 1]
```diff
@@ -0,0 +1,65 @@
+desc 'Republish all forms. Run: USER=my-email@digital.justice.gov.uk bundle exec rake republish_all_forms'
+task republish_all_forms: :environment do
+  user = User.find_by(email: ENV.fetch('USER'))
+  raise "Can not find user #{ENV.fetch('USER')}" if user.blank?
```
Comment (brenetic) [inline lib/tasks/republish.rake:4]:
```suggestion
  raise "Set 'USER'. e.g USER=darth.vadar@death-star.empire" if user.blank?
```

**Comment by brenetic** [comment]
I think we need to be able to pass in a deployment environment (`dev` || `production`). For example the in Live all the forms have already been republished in live-production. It's not a big deal to requeue them all, but having the granularity as an option i think would be useful

**Comment by tomas-stefano** [comment]
@brenetic done!

File: .circleci/config.yml [hunk 2]
```diff
@@ -101,7 +101,9 @@ workflows:
             - test
           filters:
             branches:
-              only: master
+              only:
+                - master
+                - republish-all-forms
```
Comment (brenetic) [inline .circleci/config.yml:106]: ~- republish-all-forms~

Comment (tomas-stefano) [reply .circleci/config.yml:106]: Waiting to deploy to test on https://app.circleci.com/pipelines/github/ministryofjustice/fb-publisher/814/workflows/f66a7a00-98c4-40e9-ae04-429e00a887f4 so I can remove afterwards

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**